### PR TITLE
Solved the Error 404 found on one speakers link

### PR DESCRIPTION
**Pull request by thetathya**
I have changed a error 404 problem which showed when we tried to click on Khalil AlMaawali's website.

### CODE REVIEW

**Comment by simsausaurabh** [comment]
Please raise issue for the same and reference this PR for the issue. And provide preview link for the same.

File: index.html [hunk 1]
```diff
@@ -1167,9 +1167,6 @@ <h1>Speakers</h1>
                                 <div class="image-holder">
                                     <img class="background-image" alt="Khalil AlMaawali" src="img/KhalilAlMaawali.jpg">
                                     <div class="hover-state text-center preserve3d">
-                                        <div class="social-links vertical-align">
-                                            <a target="default" href="http//fossc.om"><i class="icon fa fa-external-link"></i></a>
```
Comment (simsausaurabh) [inline index.html]: you don't have to remove the link just make it: http://fossc.om or http://fossc.om/2017/

**Comment by thetathya** [comment]
https://github.com/fossasia/2018.fossasia.org/issues/355
Sorry I forgot to mention that

**Comment by thetathya** [comment]
Done @simsausaurabh
Issue link https://github.com/fossasia/2018.fossasia.org/issues/355
Also added the link

**Comment by simsausaurabh** [comment]
Preview link ? Squash commits.

**Comment by thetathya** [comment]
Issue #355 - Error 404 when clicking on link of Khalil AlMaawali
solved

**Comment by thetathya** [comment]
Anything else @simsausaurabh  @CosmicCoder96 ?

**Comment by thetathya** [comment]
Squash Commit Done @simsausaurabh

**Comment by thetathya** [comment]
@CosmicCoder96 I have squashed my commit in one. There are other people whose commit is showing up in my pull request.

image

**Comment by hpdang** [comment]
@thetathya conflict?

**Comment by thetathya** [comment]
@hpdang Nope. I dont know whats happening. I have not even committed any changes today

**Comment by thetathya** [comment]
@hpdang Should I close this pull request?

**Comment by mariobehling** [comment]
DS store Windows files and images that exist already should not be committed. I fixed the http issue already. Cheers. Therefore closing it.

**Pull request closed without merging**